internal/cmd/database/dump.go

```diff
@@ -166,7 +179,7 @@ func dump(ch *cmdutil.Helper, cmd *cobra.Command, flags *dumpFlags, args []strin
 	// NOTE(fatih): the password is a placeholder, replace once we get rid of the proxy
 	cfg.Password = "root"
 	cfg.Address = addr.String()
-	cfg.Database = dbName
+	cfg.Database = ksName
```

Comment:
```suggestion
	cfg.Database = dbName
```

I think it's fine to keep the var as `dbName`. Reading `ksName := getDatabaseName` is likely confusing for anyone not familiar with keyspaces etc.